PGM/src/main/java/tc/oc/pgm/rotation/DynamicRotationListener.java

```diff
@@ -40,6 +40,9 @@ public void onMatchEnd(MatchEndEvent event) {
 
         // Ignore if dynamic rotations are disabled or if there is only one rotation available
         if (!config.getBoolean("dynamic", false) || rotationManager.getRotations().size() <= 1) return;
+        
+        // If a mutation was set for the next map, don't change it yet.
+        if (!MutationCommands.getInstance().getMutationQueue().isEmpty()) return;
```

Comment:
You should inject the MutationQueue

Reply:
I have only been familiar with Dependency Injection for about 12 hours so far;

Should I make a field injection or a constructor injection? (I don't know what the preference is)

Reply:
Put it in the constructor